.changeset/brown-elephants-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': major
+---
+
+added focus to alert button outline for correct specificity order
```

Comment:
this should describe how it affects the user, rather than implementation details

```suggestion
Alert: close button's focus color now matches the status color.
```